modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPReport.testcase

```diff
@@ -0,0 +1,96 @@
+@component-name = "OSB Site Initializer EVP"
+definition {
+
+	var baseURL = PropsUtil.get("portal.url");
+
+	property custom.properties = "feature.flag.LPS-169981=true";
+	property osgi.modules.includes = "osb-site-initializer-evp";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "OSB Site Initializer EVP";
+
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Create a new EVP Site") {
+			EVPUtils.addEVPSite();
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "EVP Portal");
+		}
+	}
+
+	@description = "This is a test for LPS-164731. Verify that Reports page is not visible on top menu for Employee users."
+	@priority = 4
+	test EmployeeCanNotSeeReportPage {
+		task ("Given the employee user are logged") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+		}
+
+		task ("When he goes to EVP Site") {
+			EVPUtils.gotoEVP();
+		}
+
+		task ("Then he can not see Reports Page on top menu") {
+			EVPNavigator.navItemNotPresent(navItem = "Reports");
+		}
+	}
+
+	@description = "This is a test for LPS-164731. Verify that user can access Reports page through 'Reports' Tab on top menu and it is visible to EVP Finance"
```

Comment:
Hey @joelrocha68 , Could you change the test description as previously align?



Sugestion: "LPS-164731 - Finance user should be able to see the Report card through 'Reports' Tab on top menu"